src/message/story.tsx

```diff
@@ -1,33 +1,72 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 
 import { storiesOf } from '@storybook/react'
 import { withKnobs, text, boolean } from '@storybook/addon-knobs'
 
 import Message from 'message'
+import {JSXElement} from "@babel/types"
 
 const stories = storiesOf('Message', module)
 stories.addDecorator(withKnobs)
 
+const wrapInContainer = (content: JSX.Element) => {
+  return <div style={{width: '600px', border: '1px solid lightgray'}}>
+    {content}
+  </div>
+}
+
+const veryLongMessage = "long message ".repeat(10)
+const veryLongMessageWithoutBreaks = "longmessage_".repeat(10)
+stories.add('Conversation', () => (
+    wrapInContainer(
+        <Fragment>
+          <Message messageAnnotation="Delivered - 15:29" active={false}>Msg</Message>
```

Comment:
I find the `active` props not really clear, I suppose that it means "current user" and that it changes the bubble alignement but that's a guess 😉.

Otherwise the API feels nice to use 👍.

Reply:
active is from the previous API and is not really explicit, I agree.